src/main/java/com/booking/service/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.booking.service.service.impl;
+
+import com.booking.service.dao.UserDao;
+import com.booking.service.model.User;
+import com.booking.service.service.UserService;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    private final UserDao userDao;
+
+    public UserServiceImpl(UserDao userDao) {
```

Comment:
```suggestion

    @Autowired
    public UserServiceImpl(UserDao userDao) {
```

Reply:
It still works the same. Just a visual representation that it's a Spring Bean configuration. I'll keep it my way :)